skaffold.yaml

```diff
@@ -46,5 +65,5 @@ build: &default-build
 deploy: &default-deploy
   statusCheckDeadlineSeconds: 600
   kustomize:
-    path: ./kustomize/overlay/7.0/obdemo-bank
+    path: ./kustomize/overlay/7.0/obdemo-bank/dev
```

Comment:
- Added the `prod` profile
- Update de default profile to `dev`
- The customer can add his own profile to use his own environment following the `Create a new environment` section on readme file.